spec/classes/client_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe 'clickhouse::client' do
+  context 'with defaults' do
+    it do
```

Comment:
is this really the way it is done in the community? i would have expected something like this:
```
it {
    should contain_package('clickhouse-client').with_ensure('installed')
}
```
but i am really not sure

Reply:
also below i just saw a different notation, more like i have posted. can we at least make all tests consistent?

Reply:
That is done with ruby syntax guideline. Multiline should be surrounded by `do end` and one line should be in `{}`

Reply:
But you are right and I'll make all oneliners in `{}`